lib/hashtable.cc

```diff
@@ -122,6 +122,63 @@ consume_fasta(
     delete parser;
 }
 
+void Hashtable::consume_fasta_bitsplit(
```

Comment:
Yeah, I quite like `banding`. And I think `num_batches` and `batch` are good replacements for `num_bins` and `tag`.